src/components/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+import { TodoInfo } from './TodoInfo';
+import './Todo.scss';
+
+type Props = {
+  preparedTodos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
+  <ul>
+    {preparedTodos.map(todo => (
+      <li key={todo.id} className="todo__list">
```

Comment:
it's not a list, it's one todo 

```suggestion
      <li key={todo.id} className="todo">
```